breaking: stricten types around `depends` and `invalidate` (#11201)
sveltejs/kit: .changeset/dirty-phones-report.md

---
'@sveltejs/kit': major
---

> breaking: stricten types around `depends` and `invalidate` to require a `:` when passing a string

Reviewer: it seems "stricten" is a word, but I'd never heard it before today and Chrome's spell checker is telling me it isn't a word, so maybe safer to omit it since people might just think it's a typo
```suggestion
breaking: the types for `depends` and `invalidate` now require a `:` when passing a string
```
